Add non-authoritative names for esp and ah protocols. (RPL-2700)

diff -r e935e6ab29d8 protocols
--- a/protocols	Mon Mar 02 16:04:51 2009 -0500
+++ b/protocols	Mon Mar 02 17:06:12 2009 -0500
@@ -61,7 +61,9 @@
 dsr	48	DSR		# Dynamic Source Routing Protocol
 bna	49	BNA		# BNA
 esp	50	ESP		# Encap Security Payload
+ipv6-crypt	50	IPv6-Crypt	# IPv6 Encryption Header (alt esp)
 ah	51	AH		# Authentication Header
+ipv6-auth	51	IPv6-Auth	# IPv6 Authentication Header (alt ah)
 i-nlsp	52	I-NLSP		# Integrated Net Layer Security TUBA
 swipe	53	SWIPE		# IP with Encryption
 narp	54	NARP		# NBMA Address Resolution Protocol
